From 75a914d608cfaa95210681125573fbcd92feb05c Mon Sep 17 00:00:00 2001
From: Ranjani Vaidyanathan <Ranjani.Vaidyanathan@freescale.com>
Date: Mon, 10 Mar 2014 18:04:59 -0500
Subject: [PATCH 0310/1074] ENGR00302688-1 ARM:dts:imx6sx: Fixed build break

Fixed dtb build break.

Signed-off-by: Ranjani Vaidyanathan <Ranjani.Vaidyanathan@freescale.com>
[Original patch taken from git://git.freescale.com/imx/linux-2.6-imx.git]
Signed-off-by: Biyao Zhai <biyao.zhai@windriver.com>
---
 arch/arm/boot/dts/imx6sx.dtsi |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/arm/boot/dts/imx6sx.dtsi b/arch/arm/boot/dts/imx6sx.dtsi
index 7cfb9bb..ba58671 100644
--- a/arch/arm/boot/dts/imx6sx.dtsi
+++ b/arch/arm/boot/dts/imx6sx.dtsi
@@ -904,6 +904,7 @@
 				MX6SX_PAD_QSPI1A_SCLK__ANATOP_OTG2_ID 0x17059
 			>;
 		};
+	};
 
 	usdhc1 {
 			pinctrl_usdhc1_1: usdhc1grp-1 {
-- 
1.7.5.4

